configuration/outputs.md

```diff
@@ -259,6 +259,66 @@ If enabled, the Elastic Search integration will stream analytics and insights to
         #password: "elastic search password"
 ```
 
+## Azure
+
+If enabled, the Azure integration will stream analytics and insights to an Azure endpoint.
+
+| Key | Description | Required |
+| :--- | :--- | :--- |
+| name | User defined name of this specific destination, used for mapping this destination to a workflow | Yes |
+| type | Streaming destination type \(azure\) | Yes |
+| endpoint | Azure AppInsight endpoint. | Yes |
+| api_key | Azure AppInsight API key. | Yes |
+| features | Features defines which data types stream to backend, it can be "log", "metric", "edac", "cluster", "topk" or "all". If you don't provide any value then it is all. | No |
+
+```go
+      - name: azure-integration
+        type: azure
+        endpoint: https://dc.services.visualstudio.com/v2/track
+        api_key: "Azure AppInsight api key" 
+        features: "metric"
+```
+
+## Kafka
+
+If enabled, the Kafka integration will stream analytics and insights to an Kafka endpoint.
+
+| Key | Description | Required |
+| :--- | :--- | :--- |
+| name | User defined name of this specific destination, used for mapping this destination to a workflow | Yes |
+| type | Streaming destination type \(kafka\) | Yes |
+| endpoint | Kafka broker addresses. | Yes |
+| topic | Kafka topic name. | Yes |
+| features | Features defines which data types stream to backend, it can be "log", "metric", "edac", "cluster", "topk" or "all". If you don't provide any value then it is all. | No |
+
+```go
+      - name: kafka-integration
+        type: kafka
+        endpoint: https://dc.services.visualstudio.com/v2/track
+        endpoint: <kafka broker address-1>,<kafka broker address-2>
+        topic: topic
+```
+
+## SignalFx
+
+If enabled, the SignalFx integration will stream analytics and insights to an SignalFx endpoint.
+
+| Key | Description | Required |
+| :--- | :--- | :--- |
+| name | User defined name of this specific destination, used for mapping this destination to a workflow | Yes |
+| type | Streaming destination type \(signalfx\) | Yes |
+| endpoint | SignalFx endpoint. | Yes |
+| token | SignalFx API token. | Yes |
+| features | Features defines which data types stream to backend, it can be "log", "metric", "edac", "cluster", "topk" or "all". If you don't provide any value then it is all. | No |
+
+```go
+      - name: signalfx-integration
+        type: signalfx
+        endpoint: https://ingest.us1.signalfx.com/v2
+        token: "<add signalfx api token>"
+        features: "metric,log"
+```
+
 ## Trigger Destinations
```

Comment:
looks like the section title and individual destinations have same font size (both start with double '##')
can you make individual destinations smaller (###)